tools/witx/src/docs/ast.rs

```diff
@@ -356,12 +288,10 @@ impl ToMarkdown for InterfaceFuncParam {
 impl BuiltinType {
     pub fn type_name(&self) -> &'static str {
         match self {
-            BuiltinType::String => "string",
-            BuiltinType::Char8 => "char8",
-            BuiltinType::USize => "usize",
-            BuiltinType::U8 => "u8",
+            BuiltinType::Char => "char",
+            BuiltinType::U8 { .. } => "u8",
             BuiltinType::U16 => "u16",
-            BuiltinType::U32 => "u32",
+            BuiltinType::U32 { .. } => "u32",
```

Comment:
Would it make sense to check `lang_ptr_size` here and map to `"usize"` as appropriate?